lib/matplotlib/figure.py

```diff
@@ -523,15 +542,23 @@ def contains(self, mouseevent):
         """
         Test whether the mouse event occurred on the figure.
 
-        Returns True, {}.
+        Returns
+        -------
+        Tuple of ``True``, {}.
         """
         if callable(self._contains):
             return self._contains(self, mouseevent)
         inside = self.bbox.contains(mouseevent.x, mouseevent.y)
         return inside, {}
 
     def get_window_extent(self, *args, **kwargs):
-        'get the figure bounding box in display space; kwargs are void'
+        ''''
+        Get the figure bounding box in display space; arguments are ignored.
+
+        Returns
+        -------
+        bbox
```

Comment:
Same here; I don't think the Returns table adds anything (just write "Return the figure bounding box etc.")